pkg/kratos/interfaces.go

```diff
@@ -1,14 +1,28 @@
 package kratos
 
 import (
-	hydra_client "github.com/ory/hydra-client-go/v2"
+	"context"
+	"net/http"
+
+	hClient "github.com/ory/hydra-client-go/v2"
+	kClient "github.com/ory/kratos-client-go"
 	kratos_client "github.com/ory/kratos-client-go"
 )
 
 type KratosClientInterface interface {
-	FrontendApi() kratos_client.FrontendApi
+	FrontendApi() kClient.FrontendApi
 }
 
 type HydraClientInterface interface {
-	OAuth2Api() hydra_client.OAuth2Api
+	OAuth2Api() hClient.OAuth2Api
+}
+
+type ServiceInterface interface {
+	CheckSession(context.Context, []*http.Cookie) (*kClient.Session, http.Header, error)
+	AcceptLoginRequest(ctx context.Context, identityID string, lc string) (*hClient.OAuth2RedirectTo, http.Header, error)
+	CreateBrowserLoginFlow(ctx context.Context, aal string, return_to string, login_challenge string, refresh bool, cookies []*http.Cookie) (*kClient.LoginFlow, http.Header, error)
+	GetLoginFlow(ctx context.Context, id string, cookies []*http.Cookie) (*kClient.LoginFlow, http.Header, error)
+	UpdateOIDCLoginFlow(ctx context.Context, flow string, body kClient.UpdateLoginFlowBody, cookies []*http.Cookie) (*ErrorBrowserLocationChangeRequired, http.Header, error)
+	GetFlowError(ctx context.Context, id string) (*kClient.FlowError, http.Header, error)
+	parseLoginFlowMethodBody(r *http.Request) (*kratos_client.UpdateLoginFlowBody, error)
```

Comment:
```suggestion
	AcceptLoginRequest(context.Context, string, string) (*hClient.OAuth2RedirectTo, http.Header, error)
	CreateBrowserLoginFlow(context.Context, string, string, string, bool, []*http.Cookie) (*kClient.LoginFlow, http.Header, error)
	GetLoginFlow(context.Context, string, []*http.Cookie) (*kClient.LoginFlow, http.Header, error)
	UpdateOIDCLoginFlow(context.Context, string, kClient.UpdateLoginFlowBody, []*http.Cookie) (*ErrorBrowserLocationChangeRequired, http.Header, error)
	GetFlowError(context.Context, string) (*kClient.FlowError, http.Header, error)
	parseLoginFlowMethodBody(*http.Request) (*kratos_client.UpdateLoginFlowBody, error)
```

convention goes that u dont name args in interfaces (works anyway in fairness)